config/constants.py

```diff
@@ -1,5 +1,8 @@
 from typing import Set
 
+
+DEFAULT_CATEGORY = 'Any help'
```

Comment:
This has already been added by @idisblueflash so your PR should not have this change, please, merge/rebase the latest `master`

Reply:
Yeah, bad double rebase…